src/objects/player.lua

```diff
@@ -30,13 +30,27 @@ return {
         -- resources
         self.idle = sprite.create('32x32_player.png', self.w, self.h, 0.25)
         self.walk = sprite.create('32x32_player-walk.png', self.w, self.h, 0.1)
+        self.jump = sprite.create('32x32_player-jump.png', self.w, self.h, 0.05)
+
+        self.jump.looping = false
 
         self.spr = self.walk
         self.spr:play()
 
+        self.spr = self.jump
```

Comment:
I don't think these 2 are needed -- the sprite is assigned and started when the player jumps